make sure to always process files in the same order

to get reproducible output

Index: media-player-info-22/tools/mpi2hwdb.py
===================================================================
--- media-player-info-22.orig/tools/mpi2hwdb.py
+++ media-player-info-22/tools/mpi2hwdb.py
@@ -84,5 +85,5 @@ def parse_mpi(mpi):
 #
 
 # parse MPI files
-for f in sys.argv[1:]:
+for f in sorted(sys.argv[1:]):
     parse_mpi(f)
Index: media-player-info-22/tools/mpi2udev.py
===================================================================
--- media-player-info-22.orig/tools/mpi2udev.py
+++ media-player-info-22/tools/mpi2udev.py
@@ -122,7 +122,7 @@ elif sys.argv[1] != 'udev':
     sys.exit(1)
 
 # parse MPI files
-for f in sys.argv[2:]:
+for f in sorted(sys.argv[2:]):
     parse_mpi(f, hwdb)
 
 # udev rules footer
